applications/nrf5340_audio/bin/manifest.yaml

```diff
@@ -1,7 +1,7 @@
 description: LE Audio Controller Subsystem for nRF53
-git_revision: d2a6b4853789672aa0b0eea8f88eccc267b024d9
-ll_subversion_number: '0x0CEE'
+git_revision: 7b4391eca43a690aa7f6cb1e119cdf24669f4bdc
+ll_subversion_number: '0x0CF2'
 ll_version_number: '0x0B'
-ll_subversion_number_decimal: '3310'
```

Comment:
We have to discuss this one. Controller updates must pass the internal workflow created by Frode.

Reply:
And should be done separately to any additional functionality to aid bisecting and isolating potential errors.

Reply:
Agree, I added it for interleaved, and to spark a conversation about updating the controller